.flutter-plugins

```diff
@@ -0,0 +1,8 @@
+# This is a generated file; do not edit or check into version control.
+path_provider_linux=/Users/fks/.pub-cache/hosted/pub.dartlang.org/path_provider_linux-2.1.0/
+path_provider_windows=/Users/fks/.pub-cache/hosted/pub.dartlang.org/path_provider_windows-2.0.3/
+shared_preferences=/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences-2.0.8/
+shared_preferences_linux=/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_linux-2.0.2/
+shared_preferences_macos=/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_macos-2.0.2/
+shared_preferences_web=/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_web-2.0.2/
+shared_preferences_windows=/Users/fks/.pub-cache/hosted/pub.dartlang.org/shared_preferences_windows-2.0.2/
```

Comment:
Thank you for fixed 🙆‍♂️ 
Several files have been added  folder, including this file. Are these all necessary files? If not, please remove them.